doc/quickstart.rst

```diff
@@ -14,9 +14,9 @@ obtaining it is quite easy.
 .. code:: python
 
     import numpy as np
-    
```

Comment:
Is there a compelling reason to change this file?

Reply:
Yes, it's just for fixing the mismatch between lines 60 and 68. The comment talks about training 10 epochs, but the code runs 30.

Reply:
Got it, thanks.